apps/devmo/helpers.py

```diff
@@ -206,8 +206,12 @@ def datetimeformat(context, value, format='shortdatetime', output='html'):
     Returns date/time formatted using babel's locale settings. Uses the
     timezone from settings.py
     """
+    if isinstance(value, datetime.date):
+        # Turn a date into a datetime
+        value = datetime.datetime.combine(value, datetime.datetime.min.time())
+
     if not isinstance(value, datetime.datetime):
-        # Expecting date value
+        # Expecting datetime value
```

Comment:
Need to update [a failing test](https://travis-ci.org/mozilla/kuma/builds/46384489#L2367) related to this change.